packages/core/admin/admin/src/components/CommandPalette/Group.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CommandGroup } from 'cmdk';
+import { Typography } from '@strapi/design-system';
+import { useIntl } from 'react-intl';
+
+const Group = ({ children, heading, ...rest }) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <CommandGroup
+      heading={
+        <Typography as="span" variant="sigma" textColor="neutral600">
+          {formatMessage({ id: heading })}
```

Comment:
iirc you need to set a default message otherwise in languages where it's not translated it will just show the `id`.

Because this is dynamic, you might be better off passing the formatted message as the `heading` prop?

Reply:
Can I pass a string to it ?